source/expressions.tex

```diff
@@ -3993,11 +3993,13 @@
     \terminal{::}\opt{} \terminal{delete [ ]} cast-expression
 \end{bnf}
 
-The first alternative is for non-array objects, and the second is for arrays. Whenever
-the \tcode{delete} keyword is immediately followed by empty square brackets, it shall be
-interpreted as the second alternative.\footnote{A lambda expression with a
-\grammarterm{lambda-introducer} that consists of
-empty square brackets can follow the \tcode{delete} keyword if the lambda expression is
+The first alternative is a
+\defnx{single-object delete expression}{delete!single-object}, and the
+second is an \defnx{array delete expression}{delete!array}.
+Whenever the \tcode{delete} keyword is immediately followed by empty square
+brackets, it shall be interpreted as the second alternative.\footnote{A lambda
+expression with a \grammarterm{lambda-introducer} that consists of empty square 
```

Comment:
Trailing whitespace here.

Reply:
I made a stab at fixing the whitespace issue. I'll keep an eye on the CI builder.